src/main/kotlin/no/nav/k9/los/nyoppgavestyring/query/db/SqlOppgaveQuery.kt

```diff
@@ -160,7 +163,7 @@ class SqlOppgaveQuery(
                       AND fo.ekstern_id = :feltOmrade$index
                       AND fd.ekstern_id = :feltkode$index
                       AND ov.aktiv = true
-                      AND ov.oppgavestatus in (:oppgavestatus)
+                      AND ov.oppgavestatus in (${(queryParams["oppgavestatus"] as List<Oppgavestatus>).joinToString(prefix = "'", separator = ", ", postfix = "'") { oppgavestatus -> oppgavestatus.kode }})
```

Comment:
Ble det ikke en egen liste med oppgavestatus for optimalisering av ytelse likevel?

Reply:
Det vi diskuterte på tampen i går? Dette var da på plass allerede. Skal inn med ryddesaga nå :)